### ISSUE TITLE
Is `supplier` a valid `type`?

### ISSUE THREAD

**Issue by tschmidtb51** [package-url/purl-spec#117]
I've seen many examples lately which use something like `pkg:supplier/ACME/INFUSION@1.0`. However, `supplier` is not listed in the known [PURL types](https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst) nor in the examples. Am I missing something?

**Comment by stevespringett** [comment]
No. This is not a valid purl. The NTIA Healthcare PoC has been loosely following purl syntax, however most of the strings they claim are purls will fail validation if used.

**Comment by sei-vsarvepalli** [comment]
Hello @tschmidtb51 

This is not a valid PURL. In the NTIA healthcare PoC @stevespringett  has brought it up a few times. There is much to resolve here. 

There are parts of this code left alone for demo purposes, usage of PURL is definitely one of those.

Vijay

**Comment by tschmidtb51** [comment]
@pombredanne Agreed. 

If you add a FAQ to the repo, I would suggest to add that question or include it in the examples as invalid...
